steamapi/user.py

```diff
@@ -452,24 +457,28 @@ def is_community_banned(self):
         :rtype: bool
         """
         return self._bans.CommunityBanned
+
     @cached_property(ttl=INFINITE)
     def number_of_vac_bans(self):
         """
         :rtype: int
         """
         return self._bans.NumberOfVACBans
+
```

Comment:
There's still these: (2)

Reply:
I'm not seeing any code here that is broken over multiple lines.

Are you referring to the line breaks?